pkg/apply/apply_test.go

```diff
@@ -66,41 +88,72 @@ func TestApplyBasicUpdates(t *testing.T) {
 	// Update retention and settings
 	applier.topicConfig.Spec.RetentionMinutes = 400
 	applier.topicConfig.Spec.Settings["cleanup.policy"] = "delete"
-	err = applier.Apply(ctx)
+	applier.topicConfig.Spec.Settings["max.message.bytes"] = "600000"
+	changes, err = applier.Apply(ctx)
 	require.NoError(t, err)
 	topicInfo, err = applier.adminClient.GetTopic(ctx, topicName, true)
 	require.NoError(t, err)
 
+	// test NewOrUpdatedChanges has expected shape when updating configs
+	expectedChanges = &NewOrUpdatedChanges{
+		NewChanges: nil,
+		UpdateChanges: &UpdateChangesTracker{
+			Action: "update",
+			Topic:  topicName,
+			NewConfigEntries: &[]NewConfigEntry{
+				{
+					Name:  "max.message.bytes",
+					Value: "600000",
+				},
+			},
+			UpdatedConfigEntries: &[]ConfigEntryChanges{
+				{
+					Name:    "cleanup.policy",
+					Current: "compact",
+					Updated: "delete",
+				},
+				{
+					Name:    "retention.ms",
+					Current: "30000000",
+					Updated: "27000000",
+				},
+			},
+			MissingKeys: []string{},
```

Comment:
At the end of this test there is a test case for missing keys, please add the assertion on the output there as well so we test the removal of keys.

Reply:
added some assertions here: https://github.com/getsentry/topicctl/pull/3/files#diff-12322d3d658c39e94e0f7fad6411931e5fd68ad429776d1b86ae01cbadc7af2dR144-R159